kitchen-sink-demo/src/main.ts

```diff
@@ -1,14 +1,13 @@
-import { enableProdMode } from '@angular/core';
```

Comment:
Now that we are using `main.augury.ts` these changes should probably be reverted.